### PR TITLE
Docs/known issues update

### PR DESCRIPTION
**Pull request by oblm**
#### Description
Add known issues and clarify workarounds.
#### Tests
Rendered markdown.
#### Documentation
All I do is documentation.

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -157,27 +157,25 @@ Launch the Unity editor. It should automatically detect the project but if it do
...
 
-To do this:
-
-1. Open the `FPS-SimulatedPlayerCoordinator` scene located at `Assets/Fps/Scenes`.
-2. Click on the `FPS-Start_Large` object in the [Unity Hierarchy window](https://docs.unity3d.com/Manual/Hierarchy.html), and enable the object.
-3. Open the **Navigation** pane by clicking on **Windows** > **AI** > **Navigation**.
-4. Navigate to the **Bake** tab and click on the **Bake** button.
+1. In the [Project](https://docs.unity3d.com/Manual/ProjectView.html) tab, open the `FPS-SimulatedPlayerCoordinator` scene located at `Assets/Fps/Scenes`.
+2. In the [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) tab, click on the `FPS-Start_Large` object and, in the Inspector tab, enable the object by clicking the checkbox.
+3. Open the Navigation tab by clicking **Window** > **AI** > **Navigation**.
```
Comment (ElleEss) [inline docs/get-started.md]: Open the Navigation tab; from the Unity Editor menu,  select **Window** > **AI** > **Navigation**.

(Need to re-orient the user as we were just in the Hierarchy window.

File: docs/get-started.md [hunk 2]
```diff
@@ -157,27 +157,25 @@ Launch the Unity editor. It should automatically detect the project but if it do
...
+There is a bug in the Unity Editor that stops it importing the navmesh for the `FPS-SimulatedPlayerCoordinator` correctly when you open the project. To fix this, you need to rebake the navmesh. To do this:
 
-To do this:
-
-1. Open the `FPS-SimulatedPlayerCoordinator` scene located at `Assets/Fps/Scenes`.
-2. Click on the `FPS-Start_Large` object in the [Unity Hierarchy window](https://docs.unity3d.com/Manual/Hierarchy.html), and enable the object.
-3. Open the **Navigation** pane by clicking on **Windows** > **AI** > **Navigation**.
-4. Navigate to the **Bake** tab and click on the **Bake** button.
+1. In the [Project](https://docs.unity3d.com/Manual/ProjectView.html) tab, open the `FPS-SimulatedPlayerCoordinator` scene located at `Assets/Fps/Scenes`.
+2. In the [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html) tab, click on the `FPS-Start_Large` object and, in the Inspector tab, enable the object by clicking the checkbox.
```
Comment (ElleEss) [inline docs/get-started.md]: This is a bit confusing for my tiny mind, suggest:

2. In the [Hierarchy window (Unity documentation)](https://docs.unity3d.com/Manual/Hierarchy.htm), select `FPS-Start_Large` to see it in the Inspector window. 
3. In the Inspector window, check X to enable it.  {{Editor note - what is X labelled?}}}

File: docs/get-started.md [hunk 3]
```diff
@@ -157,27 +157,25 @@ Launch the Unity editor. It should automatically detect the project but if it do
...
+3. Open the Navigation tab by clicking **Window** > **AI** > **Navigation**.
+4. Navigate to the **Bake** section and click on the **Bake** button.
```
Comment (ElleEss) [inline docs/get-started.md]: Suggest this as we are already in the Navigation window.

5. Select the **Bake** tab, and then the **Bake** button.

File: docs/known-issues.md [hunk 4]
```diff
@@ -13,3 +13,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
...
 | [Reader [Require] fields]({{urlRoot}}/content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): `Reader.<component property name>Updated` may be invoked even if `<component property name>` did not change.                                                                                 | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
+| There is a bug in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended. | 2018/10/12 | [566245](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/)         | Manually reimport shaders.                                                                  | No     |
```
Comment (ElleEss) [inline docs/known-issues.md]: Suggest:
There is a bug in the preview version of the [High Definition Render Pipeline (Unity blog)](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/)

{{ We want to have a better way to tell people they are leaving our docs when they hit certain links, but, for now, we are going with where they are headed in brackets.

File: docs/get-started.md [hunk 5]
```diff
@@ -157,27 +157,25 @@ Launch the Unity editor. It should automatically detect the project but if it do
 #### Before you start, apply these quick bugfixes:
 <%(#Expandable title="Fix Shaders")%>
 
-There is a bug in the current preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
+There is a bug in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended. We've [reported](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/) the issue to Unity.
```
Comment (jamiebrynes7) [inline docs/get-started.md]: Is this necessary info for a user at this point?

Comment (jessicafalk) [reply docs/get-started.md]: guess it helps, if people want to track the progress of a bug. though I would expect that in the known issues page and not here. Having a link to the unity forums in the get started guide might get them sidetracked

Comment (jessicafalk) [reply docs/get-started.md]: seeing as we have it also in the known issues page, would definitely say that we should remove that sentence here

**Comment by jessicafalk** [comment]
@oblm what is the state of this pr?

**Comment by oblm** [comment]
I've dealt with all of the above comments in https://github.com/spatialos/gdk-for-unity/pull/579. I'm closing this PR in favor of that one as, since this PR was opened, the files it was making changes to have been split into different, smaller files.

**Pull request closed without merging**